__tests__/kata2.booleanToWord.test.js

```diff
@@ -2,4 +2,13 @@ const { booleanToWord } = require('../src');
 
 describe('booleanToWord', () => {
   // how do we create specs again???
+
+it('returns Yes when booleanToWord is True',() => {
+  expect(booleanToWord(true)).toBe('Yes')
```

Comment:
Always be consistent with your indentation method: either go with 4 spaces or 2. I recommend 2.